### ISSUE TITLE
Incompatible with Angular 8

### ISSUE THREAD

**Issue by ghbob** [crisbeto/angular-svg-round-progressbar#187]
I tried to update to Angular 8 today and I think this time it won't work by just ignoring the warnings like I did with Angular 7 :)

```
Package "angular-svg-round-progressbar" has an incompatible peer dependency to "@angular/common" (requires "^6.0.0" (extended), would install "8.0.0").
Package "angular-svg-round-progressbar" has an incompatible peer dependency to "@angular/compiler" (requires "^6.0.0" (extended), would install "8.0.0").
Package "angular-svg-round-progressbar" has an incompatible peer dependency to "@angular/core" (requires "^6.0.0" (extended), would install "8.0.0").
Package "angular-svg-round-progressbar" has an incompatible peer dependency to "zone.js" (requires "^0.8.26", would install "0.9.1").
Package "angular-svg-round-progressbar" has an incompatible peer dependency to "core-js" (requires "^2.4.1", would install "3.1.3").
```

I tried to get a running version by updating it myself (installed angular cli, did the `ng update`, changed two tiny things in the code) but I failed at the `gulp deploy` and I currently don't have more time to spend on it.

@crisbeto any plans for an update? thanks!

**Comment by crisbeto** [comment]
I'll bump it to Angular 8 and relax the dependencies a bit once I get some more time tomorrow or over the weekend.

**Comment by crisbeto** [comment]
It's been released as 3.0.0.
